jekyll/_cci2/caching.md

```diff
@@ -197,7 +197,11 @@ Caches created via the `save_cache` step are stored for up to 15 days.
 ### Clearing cache
 {: #clearing-cache }
 
-If you need to get clean caches when your language or dependency management tool versions change, use a naming strategy similar to the previous example. Then change the cache key names in your `config.yml` file and commit the change to clear the cache.
+Caches currently cannot be cleared. If you need to generate a new set of caches because you have updated language or build management tool versions, simply update the cache key, similar to the previous example.
```

Comment:
```suggestion
Caches cannot be cleared. If you need to generate a new set of caches because you have updated language or build management tool versions, simply update the cache key, similar to the previous example.
```